### PR TITLE
feat: improve aggregator toolbox

### PR DESCRIPTION
**Pull request by benjlevesque**

### CODE REVIEW

File: packages/toolbox/src/chainlinkConversionPathTools.ts [hunk 1]
```diff
@@ -193,19 +174,12 @@ export const listAggregators = async (options?: IOptions): Promise<void> => {
...
-  console.log('#####################################################################');
-  console.log('All aggregators:');
-  console.log(allAggregators);
-  console.log('#####################################################################');
-  console.log('All aggregators nodes (currency) :');
-  console.log('../currency/src/chainlink-path-aggregators.ts');
-  console.log(aggregatorsNodesForDijkstra);
-  console.log('#####################################################################');
-  console.log('Supported currencies (advanced-logic) :');
-  console.log(
```
Comment (benjlevesque) [inline packages/toolbox/src/chainlinkConversionPathTools.ts:205]: this steop is no longer required

File: packages/toolbox/src/chainlinkConversionPathTools.ts [hunk 2]
```diff
@@ -28,54 +28,73 @@ type AggregatorUpdatedArgs = {
...
+    const lastBlock = await this.provider.getBlockNumber();
+    const chunks = chunk(
+      Array(lastBlock - this.creationBlockNumber)
+        .fill(0)
+        .map((_, i) => this.creationBlockNumber + i),
+      this.maxRange,
     );
 
+    // Get the fee proxy contract event logs
+    const logs = await Bluebird.map(
```
Comment (yomarion) [inline packages/toolbox/src/chainlinkConversionPathTools.ts:74]: Neat

**Review by yomarion — APPROVED** [review]
Super nice move

File: packages/toolbox/src/chainlinkConversionPathTools.ts (hunk 1 above)
Comment (yomarion) [reply packages/toolbox/src/chainlinkConversionPathTools.ts:205]: You've removed all the fun 🤷‍♂️

Comment (benjlevesque) [reply packages/toolbox/src/chainlinkConversionPathTools.ts:205]: that one was Darko actually ;)

File: packages/currency/src/aggregators/fantom.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (alexandre-abrioux) [inline packages/currency/src/aggregators/fantom.json:1]: Idea: also move it to a `/chainlink/` subdirectory, under `/aggregators/`

**Review by alexandre-abrioux — APPROVED** [review]
Very nice!

File: packages/currency/src/aggregators/fantom.json (hunk 3 above)
Comment (benjlevesque) [reply packages/currency/src/aggregators/fantom.json:1]: we will do that when we have other providers ;)

**Review by KolevDarko — APPROVED** [review]

**Pull request merged**